bot.py

```diff
@@ -83,6 +83,10 @@
 CHANNEL_REPORTS = "reports"
 CHANNEL_JOIN = "join-logs"
 
+# Channel IDs
+# (ideally should be either initailized on startup or done dynamically during command call rather than hardcoded)
+CHANNELID_REPORTS = 739596418762801213
+
```

Comment:
Hardcoded ID's probably shouldn't be in the code, especially since the global `CHANNEL_REPORTS` variable exists.